session1/solutions/assignment4/Rakefile

```diff
@@ -1 +1,42 @@
 # assignment 4 solution
+ #namespace :gziped_files do
+  
+  desc "Archive files to gzip to a specified path"
+  task :archive , [:M_PATH, :EXCLUDE]  do
+
+  mpath = ENV["M_PATH"]
+  dexclude = ENV["EXCLUDE"]
+
+    # Find all of the files 
+    Dir.chdir mpath
+    files = FileList["*"]
+    puts "The full file list is:"
+    files.each{|file| 
```

Comment:
What are you printing this for?

Reply:
Just for the user to see what  files he has in the folder
do i need to delete it?